docs/community/contribute.md

```diff
@@ -23,65 +23,100 @@ git clone git@github.com/YourUserName/outlines.git
 cd outlines
 ```
 
-Create a new virtual environment. *If you are using conda*:
+Create a new virtual environment:
+
+*If you are using `uv`*:
 
 ```bash
-conda env create -f environment.yml
+uv venv
+source .venv/bin/activate
+alias pip="uv pip" # ... or just remember to prepend any pip command with uv in the rest of this guide
 ```
 
-*If you are using venv*:
+*If you are using `venv`*:
 
-```python
+```bash
 python -m venv .venv
 source .venv/bin/activate
 ```
 
-Then install the dependencies in editable mode, and install the pre-commit hooks:
+*If you are using `conda`*:
+
+```bash
+conda env create -f environment.yml
+```
+
+Then install the dependencies in editable mode, and install the `pre-commit` hooks:
 
-```python
+```bash
 pip install -e ".[test]"
 pre-commit install
 ```
```

Comment:
```suggestion
```bash
pip install -e ".[test]"
pre-commit install
```

Outlines provides optional dependencies for different supported backends, which you can install with

```bash
pip install ".[vllm]"
```

A list of supported optional dependencies can be found in the [installation guide](/installation).
```

I _think_ that's the right relative link.